portal/portal-render-engine-impl/impl/src/webapp/vm/trinity/includePageNav.vm

```diff
@@ -63,9 +63,9 @@
 
 	<div id="toolSubsitesContainer">
 		<nav id="toolMenu" role="navigation" aria-label="${rloader.sit_toolshead}" class="Mrphs-toolsNav__menu">
-			<h2 class="skip" tabindex="-1" id="totoolmenu">${rloader.sit_toolshead}</h2>
+			<h2 class="skip fs-5 text" tabindex="-1" id="totoolmenu">${rloader.sit_toolshead}</h2>
```

Comment:
Doesn't need to be added to this PR but seeing skip on this line reminded me of https://getbootstrap.com/docs/5.1/getting-started/accessibility/#visually-hidden-content which I think is another change we'll have to make?